[docs]: update examples + browser config (#777)
browserbase/stagehand: docs/reference/initialization_config.mdx

// Custom configuration
const stagehand = new Stagehand({
	env: "LOCAL",

> 	// env: "BROWSERBASE", // For remote Browser sessions (needs API keys)

Reviewer: ```suggestion
	// env: "BROWSERBASE", // To run remotely on Browserbase (needs API keys)
```

	verbose: 1,
	enableCaching: true,
	logger: (logLine: LogLine) => {
		console.log(`[${logLine.category}] ${logLine.message}`);
	},
    // LLM configuration

>     modelName: "name-of-model", /* Name of the model to use */

Reviewer: ```suggestion
    modelName: "google/gemini-2.0-flash", /* Name of the model to use in "provider/model" format */
```

        <ResponseField name="path" type="string">
          File path for the HAR.
        </ResponseField>

>         <ResponseField name="mode" type='"full" | "minimal"'>

Reviewer: does this work

          Mode for HAR recording.
        </ResponseField>
        <ResponseField name="omitContent" type="boolean">
          Whether to omit content in HAR recording.
        </ResponseField>

>         <ResponseField name="content" type='"omit" | "embed" | "attach"'>

Reviewer: will this work?
